test/finufftGuru_test.cpp

```diff
@@ -171,36 +171,28 @@ int main(int argc, char* argv[])
   } else
     printf("\texec \t\t\t\t\t%.3g s\n", exec_t);
 
-  timer.restart();                     // Guru Step 4
-  FINUFFT_DESTROY(plan);
-  double destroy_t = timer.elapsedsec();
-  printf("\tdestroy\t\t\t\t\t%.3g s\n", destroy_t);
-  
-  double totalTime = plan_t + sort_t + exec_t + destroy_t;
+  double totalTime = plan_t + sort_t + exec_t;
   if (type!=3)
     printf("ntr=%d: %lld NU pts to %lld modes in %.3g s \t%.3g NU pts/s\n", ntransf, (long long)M,(long long)N, totalTime, ntransf*M/totalTime);
   else
     printf("ntr=%d: %lld NU pts to %lld NU pts in %.3g s \t%.3g tot NU pts/s\n", ntransf, (long long)M,(long long)N, totalTime, ntransf*(N+M)/totalTime);
 
-  
   // Comparing timing results with repeated calls to corresponding finufft function...
-  FFTW_CLEANUP();
-  FFTW_CLEANUP_THREADS();
-  FFTW_FORGET_WISDOM();
+  //FFTW_CLEANUP();
```

Comment:
Remove? Any need to keep commented-out code?

Reply:
this test code is a bit messy, but persists as a general command-line guru driver. These FFTW commands are left to remind us that Andrea found they were needed to get correct timings for the next finufft call.

Reply:
All right. So can we uncomment them? If not, maybe leave an explanatory comment about timings?

Reply:
Commented. I declare this no longer part of the main testers, unsupported.

Reply:
Commented. I declare this an unsupported test code and will move to perftests which is more for devs.